src/listener.ts

```diff
@@ -139,6 +139,14 @@ const responseViaResponseObject = async (
         done = true
       })
       if (!chunk) {
+        if (i === 1 && resHeaderRecord['transfer-encoding'] !== 'chunked') {
```

Comment:
Is it better to add the other stream-styled headers like these?

* `x-accel-buffering: no`
* `content-type: text/event-stream`

Reply:
Thank you for your comment.

When `x-accel-buffering: no` or `content-type: text/event-stream` is specified, `transfer-encoding: chunked` is usually included, so I think `resHeaderRecord[‘transfer-encoding’] !== ‘chunked’` alone is sufficient.

https://github.com/honojs/hono/blob/530ab09ae10caf33903dfb677dff239df01d5ded/src/helper/streaming/sse.ts#L86-L87

Additionally, even if this condition check is not perfect, the performance degradation is minimal, so I believe it will not cause issues in the asynchronous response pattern.